docs/src/views/Colors.tsx

```diff
@@ -0,0 +1,352 @@
+            enough to define those that are actually used in the app.
+          </p>
+          <p>
+            Below there is an example of base tints of Teams' natural colors provided. Full list of
+            all gradient values for them can be found on{' '}
+            {link('Teams color palette page', '/color-palette')}.
+          </p>
+          <Grid columns={2} variables={{ gridGap: '2rem' }}>
+            {_.map(naturalColors, (variants, color) => (
+              <div key={color}>
+                <ColorBox
+                  name={color}
+                  rounded
+                  value={colors[color][600]}
+                  copyToClipboardIcon={false}
+                />
+              </div>
+            ))}
+          </Grid>
+
+          <Header as="h3">Contextual colors</Header>
+          <p>
+            This part of the palette may include {code('brand')} color as well as {code('danger')},{' '}
+            {code('success')}, {code('info')} colors, etc.
+          </p>
+
+          <Grid columns={2}>
+            {_.map(contextualColors, (variants, color) => (
+              <div key={color}>
+                <ColorBox
+                  name={color}
+                  rounded
+                  size="big"
+                  value={colors[color][600]}
+                  copyToClipboardIcon={false}
+                />
+              </div>
+            ))}
+          </Grid>
+
+          <Header as="h3">All colors</Header>
+          <p>
+            If the theme requires more colors, they can be added to color palette as needed. These
+            are all colors available in the Teams' theme color palette.
+          </p>
+          <Grid columns={2} variables={{ gridGap: '2rem' }}>
+            {_.map(['black', 'white'], color => (
+              <div key={color}>
+                <ColorBox
+                  name={color}
+                  size="big"
+                  value={colors[color]}
+                  copyToClipboardIcon={false}
+                />
+              </div>
+            ))}
+            {_.map(
+              { ...contextualColors, ...naturalColors, ...transparentColors },
+              (variants, color) => (
+                <div key={color}>
+                  <ColorVariants name={color} headerOnly />
+                </div>
+              ),
+            )}
+          </Grid>
+          <p>
+            To see all colors variants in the palette, follow this{' '}
+            <Text as={Link} weight="bold" content="link" color="brand" to="color-palette" />.
+          </p>
+
+          <Header as="h2" content="Color scheme" />
+          <p>
+            Now that we know how the color palette is defined, let's see how we can use these
+            values, so that they will work correctly when different themes will be applied. As part
+            of each theme, we are defining <b>color scheme</b>, which will define the design tokens
```

Comment:
```diff
-Now that we know how the color palette is defined, let's see how we can use these
-values, so that they will work correctly when different themes will be applied. As part
-each theme, we are defining <b>color scheme</b>, which will define the design tokens
+Each our theme defines <b>color scheme</b>
```

Red text in diff looks useless...